src/content/blog/astro-140.mdx

```diff
@@ -16,7 +16,7 @@ We just released Astro 1.4.0, featuring:
 
 - **[Astro.cookies](#astro-cookies)**: A simplified approach to manipulating cookies within Astro pages and API routes.
 - **[Support for strict dependency installation](#strict-dependency-installation)**: pnpm users can now have the benefits of its default semi-strict dependency installation strategy in Astro projects.
-- **[Better control over style ordering](#style-ordering)**: We've fixed style ordering so that styles defined in Astro components are lower in bundled CSS than global styles you import.
+- **[Better control over style ordering](##better-control-over-style-ordering)**: We've fixed style ordering so that styles defined in Astro components are lower in bundled CSS than global styles you import.
```

Comment:
```suggestion
- **[Better control over style ordering](#better-control-over-style-ordering)**: We've fixed style ordering so that styles defined in Astro components are lower in bundled CSS than global styles you import.
```